lib/test-collection.js

```diff
@@ -124,14 +125,14 @@ class TestCollection extends EventEmitter {
 			context = null;
 		}
 
-		return new Test(hook.metadata, title, hook.fn, context, this._emitTestResult);
+		return new Test(hook.metadata, title, hook.fn, false, context, this._emitTestResult);
```

Comment:
Yea I was having my doubts about this. On the one hand it's an internal API so hey, but on the other it's a lot of arguments…